examples/pytorch/ogb/ngnn/README.md

```diff
@@ -0,0 +1,387 @@
+
+### ogbl-ddi
+
+#### performance
+
+<table>
+   <tr>
+      <th></th>
+      <th colspan=3 style="text-align: center;">test set</th>
+      <th colspan=3 style="text-align: center;">validation set</th>
+      <th>#parameters</th>
+   </tr>
+   <tr>
+      <td></td>
+      <td>Hits@20</td>
+      <td>Hits@50</td>
+      <td>Hits@100</td>
+      <td>Hits@20</td>
+      <td>Hits@50</td>
+      <td>Hits@100</td>
+      <td></td>
+   </tr>
+   <tr>
+      <td>GCN(ogb)</td>
+      <td>37.07% ± 5.07%</td>
+      <td>\</td>
+      <td>\</td>
+      <td>55.50% ± 2.08%</td>
+      <td>\</td>
+      <td>\</td>
+      <td rowspan=1>1,289,985</td>
+   </tr>
+   <tr>
+      <td>GCN(paper)</td>
+      <td>47.82% ± 5.90%</td>
+      <td>79.56% ± 3.83%</td>
+      <td>87.58% ± 1.33%</td>
+      <td>63.19% ± 0.97%</td>
+      <td>68.31% ± 0.18%</td>
+      <td>70.89% ± 0.12%</td>
+      <td>1,355,777</td>
+   </tr>
+   <tr>
+      <td>GCN+NGNN(paper)</td>
+      <td>48.22% ± 7.00%</td>
+      <td>82.56% ± 4.03%</td>
+      <td>89.48% ± 1.68%</td>
+      <td>65.95% ± 1.16%</td>
+      <td>70.24% ± 0.50%</td>
+      <td>72.54% ± 0.62%</td>
+      <td rowspan=2>1,487,361</td>
+   </tr>
+   <tr>
+      <td>GCN+NGNN(ours; 50runs)</td>
+      <td><b>54.83% ± 15.81%</b></td>
+      <td><b>93.15% ± 2.59%</b></td>
+      <td><b>97.05% ± 0.56%</b></td>
+      <td>71.21% ± 0.38%</td>
+      <td>73.55% ± 0.25%</td>
+      <td>76.24% ± 1.33%</td>
+   </tr>
+   <tr>
+      <td>GraphSage(ogb)</td>
+      <td>53.90% ± 4.74%</td>
+      <td>\</td>
+      <td>\</td>
+      <td>62.62% ± 0.37%</td>
+      <td>\</td>
+      <td>\</td>
+      <td rowspan=1>1,421,057</td>
+   </tr>
+   <tr>
+      <td>GraphSage(paper)</td>
+      <td>54.27% ± 9.86%</td>
+      <td>82.18% ± 4.00%</td>
+      <td>91.94% ± 0.64%</td>
+      <td>67.54% ± 0.75%</td>
+      <td>71.07% ± 0.22%</td>
+      <td>72.82% ± 0.23%</td>
+      <td>1,486,849</td>
+   </tr>
+   <tr>
+      <td>GraphSage+NGNN(paper)</td>
+      <td>60.75% ± 4.94%</td>
+      <td>84.58% ± 1.89%</td>
+      <td>92.58% ± 0.88%</td>
+      <td>68.05% ± 0.68%</td>
+      <td>71.14% ± 0.33%</td>
+      <td>72.77% ± 0.09%</td>
+      <td rowspan=2>1,618,433</td>
+   </tr>
+   <tr>
+      <td>GraphSage+NGNN(ours; 50runs)</td>
+      <td>57.70% ± 15.23%</td>
+      <td><b>96.18% ± 0.94%</b></td>
+      <td><b>98.58% ± 0.17%</b></td>
+      <td>73.23% ± 0.40%</td>
+      <td>87.20% ± 5.29%</td>
+      <td>98.71% ± 0.22%</td>
+   </tr>
+</table>
+
+#### Reproduction of performance
+
+- GCN + NGNN
+```{.bash}
+python main.py --dataset ogbl-ddi --device 0 --ngnn_type input --epochs 800 --dropout 0.5 --num_layers 2 --lr 0.0025 --batch_size 16384 --runs 50
+```
+
+- GraphSage + NGNN
+```{.bash}
+python main.py --dataset ogbl-ddi --device 1 --ngnn_type input --use_sage --epochs 600 --dropout 0.25 --num_layers 2 --lr 0.0012 --batch_size 32768 --runs 50
```

Comment:
Do we still need scripts then?

Reply:
I think so. It follows the rules of ogb (https://ogb.stanford.edu/docs/leader_overview/  --> How Leaderboards Work? -->code --> Please add README and provide enough instruction **(i.e., exact command)** to reproduce the submitted result.)

Reply:
Yes, but the command with `python main.py ...` seems to be a duplicate of the `.sh` files?

Reply:
Is it a good solution to remove the ```scripts``` folder(and its contents) and related description in ```README.md```?

Reply:
Yes, I think so.